src/components/Reports/AdditionalSalaryRequest/CompleteForm/CompleteForm.tsx

```diff
@@ -42,21 +31,35 @@ interface CompleteFormValues {
   housingDownPayment: string;
   autoPurchase: string;
   reimbursableExpenses: string;
+  defaultPercentage: boolean;
 }
 
 export const CompleteForm: React.FC = () => {
   const { t } = useTranslation();
-  const locale = useLocale();
-  const theme = useTheme();
   const { selectedSection } = useAdditionalSalaryRequest();
-
-  const categories = useCompleteFormCategories();
-
+  const theme = useTheme();
   const name = 'Doc, John';
   const accountNumber = '00123456';
   const primaryAccountBalance = 20307.58;
   const remainingAllowableSalary = 17500.0;
 
+  const createCurrencyValidation = useCallback(
+    (fieldName: string, max?: number) => {
+      let schema = yup
+        .number()
+        .min(0, t('{{field}} amount must be positive', { field: fieldName }))
```

Comment:
Would it make sense to start with `amount` from `yupHelpers.ts` and then add required and max checks?

```suggestion
      let schema = amount(fieldName)
```